consensus/consensus_v2.go

```diff
@@ -736,6 +737,26 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 	return
 }
 
+// LastCommitSig returns the byte array of aggregated commit signature and bitmap of last block
+func (consensus *Consensus) LastCommitSig() ([]byte, []byte, error) {
+	if consensus.blockNum <= 1 {
+		return nil, nil, nil
+	}
+	msgs := consensus.PbftLog.GetMessagesByTypeSeq(msg_pb.MessageType_COMMITTED, consensus.blockNum-1)
```

Comment:
Add a todo to put commit sigs into db, instead of memory.

Reply:
This should be OK, since I periodically clean up the memory